helpers.go

```diff
@@ -60,7 +61,7 @@ func (inst *ParserInstance) FindProxy(urlString string) (bool, string) {
 	req := new(findProxyRequest)
 	req.inst = inst
 	req.url = u.String()
-	req.host = u.Host
+	req.host = strings.Split(u.Host, ":")[0]
 	req.resp = make(chan *parserResponse, 1)
```

Comment:
Could we use `https://golang.org/pkg/net/#SplitHostPort` here instead?

Reply:
I see your point but It's a bit tricky to use that here. I used https://golang.org/pkg/net/url/#URL.Hostname instead.

Reply:
Works for me, thank you!

Reply:
My pleasure. Thanks for upstreaming this. I can finally delete my fork!